stock_scheduler_assignation_horizon/tests/test_res_config_settings.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from datetime import date, timedelta
+
+from .test_res_config_settings_common import ResConfigSettingsCase
+
+
+class DeliverySchedulerLimitCase(ResConfigSettingsCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+    def test_delivery_scheduler_horizon_limit(self):
+        self.settings.is_moves_assignation_limited = True
+        self.settings.moves_assignation_horizon = 2
+        self.assertEqual(self.company.is_moves_assignation_limited, True)
```

Comment:
here you are testing the res.config.settings relation: split this to another test